dockerfiles/base/scripts/base/startup_03_pre_networking.sh

```diff
@@ -87,7 +87,15 @@ get_image_manifest() {
 
   # Load images from file
   BOOTSTRAP_IMAGE_LIST=$(cat ${SCRIPTS_BASE_CONTAINER_SOURCE_DIR}/images/images-bootstrap)
-  IMAGE_LIST=$(cat /version/$1/images)
+  if [ -z "${CHE_SINGLE_PORT:-}" ]; then
+    IMAGE_LIST=$(cat /version/$1/images | sed '/IMAGE_TRAEFIK/d')
+        if [ -z "${CHE_MULTIUSER:-}" ]; then
```

Comment:
is that this IF should be inside of CHE_SINGLE_PORT or it's not related ?

Reply:
I first check if it's single port and define image list accordingly. Do you mean it should not be if in if? but two if clauses?

Reply:
yes I was wondering why the multiuser is part of the singleport condition
if !single_port
   remove traefik image

if !multiuser
  remove multiuser images

for me they are separated